agent/agent.go

```diff
@@ -1162,7 +1162,7 @@ func (a *agent) createTailnet(ctx context.Context, agentID uuid.UUID, derpMap *t
 		return nil, err
 	}
 
-	reconnectingPTYListener, err := network.Listen("tcp", ":"+strconv.Itoa(codersdk.WorkspaceAgentReconnectingPTYPort))
+	reconnectingPTYListener, err := network.Listen("tcp", ":"+strconv.Itoa(workspacesdk.WorkspaceAgentReconnectingPTYPort))
```

Comment:
We should remove all the "Workspace" prefixes since it's the workspacesdk

Reply:
goooooood idea